src/Sulu/Bundle/WebsiteBundle/ReferenceStore/WebspaceReferenceStore.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\ReferenceStore;
+
+use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
+
+class WebspaceReferenceStore implements ReferenceStoreInterface
+{
+    const WEBSPACE_REFERENCE_ALIAS = 'webspace';
```

Comment:
wouldnt it be easier to provide a static method that returns the tag for a given `webspackeKey` instead of building the tag using this constant in every service?